### PR TITLE
julia: update mentoring notes

### PR DESCRIPTION
**Pull request by cmcaine**
Changes:

839b411 (Colin Caine, 36 seconds ago)
   julia: fix atbash-cipher typo

9e8a5b9 (Colin Caine, 86 seconds ago)
   julia: add raindrops mentoring notes

### CODE REVIEW

**Comment by cmcaine** [comment]
When this is merged we will have mentoring notes for all core track exercises :tada:

**Review by SaschaMann — APPROVED** [review]
Looks good!

File: tracks/julia/exercises/raindrops/mentoring.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    noises = (3 => "Pling", 5 => "Plang", 7 => "Plong")
+    acc = ""
+    for (factor, noise) in noises
+        n % factor == 0 && (acc *= noise)
+    end
+    isempty(acc) ? string(n) : acc
+end
+```
+
+pdb5627's solution
```
Comment (SaschaMann) [inline tracks/julia/exercises/raindrops/mentoring.md:43]: Do you have a link to it?

Comment (cmcaine) [reply tracks/julia/exercises/raindrops/mentoring.md:43]: Nope it's not published yet

**Pull request merged**